reqhandler/request_handler_test.go

```diff
@@ -28,3 +29,46 @@ func (s *RequestHandlerSuite) TestDataReadUDPConnectionHasUDPPacketSize(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(len(output), Equals, 512)
 }
+
+// this can be configurable for specific tests
+func getTestHeaders() map[FieldName][]byte {
+	return map[FieldName][]byte{
+		ID:       []byte{0x00, 0x01},
+		QUERY:    []byte{0x00, 0x01},
+		OPCODE:   []byte{0x00, 0x01},
+		AUTHANS:  []byte{0x00, 0x01},
+		TRUNC:    []byte{0x00, 0x01},
+		RDESC:    []byte{0x00, 0x01},
+		RAVAIL:   []byte{0x00, 0x01},
+		RESPCODE: []byte{0x00, 0x00},
+		QDCOUNT:  []byte{0x00, 0x01},
+		ANCOUNT:  []byte{0x00, 0x01},
+		NSCOUNT:  []byte{0x00, 0x01},
+		ARCOUNT:  []byte{0x00, 0x01},
+	}
+}
+
+func buildTestHeaders() ([]byte, map[FieldName][]byte) {
+	h := getTestHeaders()
+	data := make([]byte, 12)
+	data[0] = h[ID][0]
+	data[1] = h[ID][1]
+	data[2] = byte(uint(h[QUERY][1]) << (8 - HeaderFieldLengths[QUERY]))
+	return data, h
```

Comment:
What kind of Header is this function supposed to create? It isn't clear from the function name.